docs/Documentation-CHN/SystemDesign/0-Architecture/1-Architecture.md

```diff
@@ -33,23 +33,8 @@ IoTDB 采用客户端-服务器架构，如下图所示。
 
 其中服务器端主要包括查询引擎，用来处理用户的所有请求，并分发到对应的管理组件，包括数据写入层、数据查询、元数据管理、权限管理等模块。
 
-* [数据文件](../1-TsFile/1-TsFile.md)
-* [查询引擎](../2-QueryEngine/1-QueryEngine.md)
-* [元数据管理](../3-SchemaManager/1-MManager.md)
-* [存储引擎](../4-StorageEngine/1-StorageEngine.md)
-* [数据查询](../5-DataQuery/1-DataQuery.md)
-* [权限管理](../6-Administration/1-Administration.md)
-
-## 连接器
-
-IoTDB 与大数据系统进行了对接。
-
-* [Hadoop-TsFile](../7-Connector/1-Hadoop-TsFile.md)
-* [Hive-TsFile](../7-Connector/2-Hive-TsFile.md)
-* [Spark-TsFile](../7-Connector/3-Spark-TsFile.md)
-* [Spark-IoTDB](../7-Connector/4-Spark-IoTDB.md)
-* [Grafana](../7-Connector/5-Grafana.md)
-
-## 工具
-
-* [同步工具](../8-Tools/1-Sync.md)
\ No newline at end of file
+* [数据文件](https://iotdb.apache.org/#/SystemDesign/progress/chap1/sec1)
```

Comment:
```suggestion
* [数据文件](/#/SystemDesign/progress/chap1/sec1)
```

Reply:
it seems that this could not link to the local file